src/utils/service.zig

```diff
@@ -186,7 +187,7 @@ pub const ReturnHandler = struct {
 /// It's guaranteed to run at least once in order to not race initialization with
 /// the `exit` flag.
 pub fn runService(
-    logger: Logger,
+    logger: anytype,
```

Comment:
Every log message already specifies the name of the service, so this function could just use ServiceManager as its scope.

If you think the service name should really be part of the scope, that's fine, but then I would make it something like "ServiceManager.\<service name\>" and also remove the service name from each of the log messages. Also you could accept Logger as the input here, and add the scope within this function.

Reply:
[81aab5c](https://github.com/Syndica/sig/pull/345/commits/81aab5c850faeafb73b0727b3e575e97917f4a2b)